packages/table/package.json

```diff
@@ -8,6 +8,8 @@
   "unpkg": "dist/table.bundle.js",
   "scripts": {
     "build:gulp": "$(cd ..; npm bin)/gulp tsc-table sass-table",
```

Comment:
Yeah, wasn't sure if running in parallel would be safe, so wanted to get your thoughts before playing with that.